### PR TITLE
Update QCAD.app to v3.12.4

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Casks/qcad.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'qcad' do
-  version '3.9.8'
-  sha256 '7e6f04d7989914bb3543f2eb64265453d9f26c98d7bd671b30bd71cf3ab07677'
+  version '3.12.4'
+  sha256 'd2342b8d70e8d1e896698f9adce9502ef8efc5dbe7ba083eefbb2e3e1d5ad106'
 
-  url "http://www.qcad.org/archives/qcad/qcad-#{version}-osx-10.6-10.10.dmg"
+  url "http://www.ribbonsoft.com/archives/qcad/qcad-#{version}-osx-10.6-10.11.dmg"
```
Comment (vitorgalvao) [inline Casks/qcad.rb]: Where are you getting this from?

Comment (adityadalal924) [inline Casks/qcad.rb]: No idea, the officeal website still shos a url in the form:

`http://qcad.org/archives/qcad/qcad-3.12.4-osx-10.6-10.11.dmg`

Comment (vitorgalvao) [inline Casks/qcad.rb]: Yep, that’s what I’m seeing as well.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 25c8d9dec1d7fd83fd6b3d31bb9948ef981438dd. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**